### PR TITLE
fix(engineer): add option --featureDiscoveryRoot 

### PR DESCRIPTION
**Pull request by cylon3035**
Add option --featureDiscoveryRoot to specify root folder for feature discovery within the feature package

### CODE REVIEW

File: packages/engineer/src/cli.ts [hunk 1]
```diff
@@ -42,6 +42,7 @@ program
     .option('--title <title>', 'application title to display in browser')
     .option('--publicConfigsRoute <publicConfigsRoute>', 'public route for configurations')
     .option('--engineerEntry <engineerEntry>', 'entry feature for engineer', 'engineer/gui')
+    .option('--featureDiscoveryRoot <featureDiscoveryRoot>', 'package subdirectory where feature discovery starts')
```
Comment (AdamYahid) [inline packages/engineer/src/cli.ts:45]: Needs to have a default value ('.') and also add to build script below

File: packages/engineer/test/dev-server.unit.ts [hunk 2]
```diff
@@ -34,6 +35,7 @@ describe('engineer:dev-server', function () {
         overrideConfig = [],
         outputPath,
         runtimeOptions = {},
+        featureDiscoveryRoot = undefined,
```
Comment (AdamYahid) [inline packages/engineer/test/dev-server.unit.ts:38]: undefined is the default value anyway, no need to be explicit

File: packages/engineer/test/fixtures/engine-feature-roots/package.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+  "name": "@fixture/engine-single-feature",
+  "version": "14.3.1",
```
Comment (AdamYahid) [inline packages/engineer/test/fixtures/engine-feature-roots/package.json:3]: Bump to 16.0.0

**Review by AdamYahid — CHANGES_REQUESTED** [review]
Awesome work, great job. See comment.

File: packages/engineer/test/fixtures/engine-feature-roots/package.json (hunk 3 above)
Comment (cylon3035) [reply packages/engineer/test/fixtures/engine-feature-roots/package.json:3]: I assume all test fixtures, right?

File: packages/engineer/src/cli.ts (hunk 1 above)
Comment (cylon3035) [reply packages/engineer/src/cli.ts]: Added, thanks.

File: packages/engineer/test/fixtures/engine-feature-roots/package.json (hunk 3 above)
Comment (cylon3035) [reply packages/engineer/test/fixtures/engine-feature-roots/package.json]: I think yes, also fixed copypasted package name 😅

**Review by AviVahl — APPROVED** [review]

**Pull request merged**